build-system/pr-check/utils.js

```diff
@@ -287,70 +253,69 @@ function uploadOutput_(containerDirectory, gcloudOutputFileName, outputDirs) {
 }
 
 /**
- * Zips and uploads the build output to a remote storage location
+ * Stores unminified build files to the CI workspace.
  */
-function uploadUnminifiedOutput() {
-  uploadOutput_(
-    UNMINIFIED_CONTAINER_DIRECTORY,
-    UNMINIFIED_GCLOUD_OUTPUT_FILE,
-    BUILD_OUTPUT_DIRS
-  );
+function storeUnminifiedBuildToWorkspace() {
+  storeBuildToWorkspace_(UNMINIFIED_CONTAINER_DIRECTORY);
 }
 
 /**
- * Zips and uploads the nomodule output to a remote storage location
+ * Stores nomodule build files to the CI workspace.
  */
-function uploadNomoduleOutput() {
-  uploadOutput_(
-    NOMODULE_CONTAINER_DIRECTORY,
-    NOMODULE_GCLOUD_OUTPUT_FILE,
-    APP_SERVING_DIRS
-  );
+function storeNomoduleBuildToWorkspace() {
+  storeBuildToWorkspace_(NOMODULE_CONTAINER_DIRECTORY);
 }
 
 /**
- * Zips and uploads the module output to a remote storage location
+ * Stores module build files to the CI workspace.
  */
-function uploadModuleOutput() {
-  uploadOutput_(
-    MODULE_CONTAINER_DIRECTORY,
-    MODULE_GCLOUD_OUTPUT_FILE,
-    APP_SERVING_DIRS
-  );
+function storeModuleBuildToWorkspace() {
+  storeBuildToWorkspace_(MODULE_CONTAINER_DIRECTORY);
 }
 
 /**
- * Zips and uploads the output for the given experiment to a remote storage
- * location
- * @param {string} exp
+ * Stores an experiment's build files to the CI workspace.
+ * @param {string} exp one of 'experimentA', 'experimentB', or 'experimentC'.
  */
-function uploadExperimentOutput(exp) {
-  uploadOutput_(exp, EXPERIMENT_GCLOUD_OUTPUT_FILE(exp), APP_SERVING_DIRS);
+function storeExperimentBuildToWorkspace(exp) {
+  storeBuildToWorkspace_(exp);
 }
 
 /**
- * Replaces URLS in HTML files, zips and uploads nomodule output,
- * and signals to the AMP PR Deploy bot that the upload is complete.
+ * Replaces URLS in HTML files, compresses and stores nomodule build in CI artifacts.
  */
-async function processAndUploadNomoduleOutput() {
+async function storeBuildToArtifacts() {
   await replaceUrls('test/manual');
   await replaceUrls('examples');
-  uploadNomoduleOutput();
+
+  if (isCircleciBuild()) {
```

Comment:
Since this function should be a no-op if called outside CircleCI, maybe rewrite it as follows for clarity?

```js
async function processAndStoreBuildToArtifacts() {
  if (!isCircleciBuild()) {
    return;
  }
  await replaceUrls('test/manual');
  await replaceUrls('examples');
  // Do other stuff
}
```

Reply:
Done